client/layout/index.jsx

```diff
@@ -166,7 +166,7 @@ Layout = React.createClass( {
 					isEnabled={ translator.isEnabled() }
 					isActive={ translator.isActivated() } />
 				{ this.renderPreview() }
-				{ config.isEnabled( 'happychat' ) && this.props.chatIsOpen && <AsyncLoad require="components/happychat" /> }
+				{ config.isEnabled( 'happychat' ) && this.props.chatIsOpen && <AsyncLoad require="extensions/happychat/components" /> }
```

Comment:
Referencing an extension from the core Calypso code breaks the contract and spirit of an extension. This will need to be moved to the extension code itself before this work can truly be considered complete.